Blockzilla/BrowserViewController.swift

```diff
@@ -358,6 +363,33 @@ class BrowserViewController: UIViewController {
 
         Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.click, object: TelemetryEventObject.eraseButton)
     }
+    
+    private func requestReviewIfNecessary() {
+        guard #available(iOS 10.3, *) else { return }
+
+        let currentLaunchCount = UserDefaults.standard.integer(forKey: BrowserViewController.userDefaultsLaunchCountKey)
+        let threshold = UserDefaults.standard.integer(forKey: BrowserViewController.userDefaultsLaunchThresholdKey)
+
+        if threshold == 0 {
+            UserDefaults.standard.set(14, forKey: BrowserViewController.userDefaultsLaunchThresholdKey)
+        }
+
+        if currentLaunchCount <= threshold {
+            return
+        }
+
+        SKStoreReviewController.requestReview()
+
+        // Increment the threshold by 50 so the user is not constantly pestered with review requests
+        switch threshold {
+            case 14:
+                UserDefaults.standard.set(64, forKey: BrowserViewController.userDefaultsLaunchThresholdKey)
+            case 64:
+                UserDefaults.standard.set(114, forKey: BrowserViewController.userDefaultsLaunchThresholdKey)
+            default:
+                break
+        }
+    }
```

Comment:
I'll have to go back and read the issue specs, I am surprised this code isn't one-time only.